gcloud-java-contrib/gcloud-java-nio-examples/src/main/java/com/google/gcloud/nio/examples/ListFilesystems.java

```diff
@@ -43,4 +43,5 @@ private static void listFilesystems() {
       System.out.println("  " + p.getScheme());
     }
   }
+
 }
```

Comment:
Thanks for the context, I learned something. I pushed an update with the newline but it doesn't seem to show up on GitHub yet, not sure why.